src/main/java/com/github/onsdigital/babbage/configuration/Babbage.java

```diff
@@ -15,19 +15,13 @@ public class Babbage implements AppConfig {
     // env var keys
     private static final String API_ROUTER_URL = "API_ROUTER_URL";
     private static final String DEV_ENVIRONMENT_KEY = "DEV_ENVIRONMENT";
-    private static final String ENABLE_CACHE_KEY = "ENABLE_CACHE";
-    private static final String ENABLE_METRICS_KEY = "ENABLE_METRICS";
-    private static final String METRICS_FORMAT_KEY = "METRICS_FORMAT";
     private static final String ENABLE_NAVIGATION_KEY = "ENABLE_NAVIGATION";
-    private static final String ENABLE_LEGACY_CACHE_API = "ENABLE_LEGACY_CACHE_API";
     private static final String HIGHCHARTS_EXPORT_SERVER_KEY = "HIGHCHARTS_EXPORT_SERVER";
     private static final String IS_PUBLISHING_KEY = "IS_PUBLISHING";
     private static final String MATHJAX_EXPORT_SERVER_KEY = "MATHJAX_EXPORT_SERVER";
     private static final String MAXAGE_SERVICE_KEY = "MAXAGE_SERVER";
     private static final String MAX_CACHE_ENTRIES = "CACHE_ENTRIES";
     private static final String MAX_OBJECT_SIZE = "CACHE_OBJECT_SIZE";
-    private static final String POST_PUBLISH_CACHE_MAX_AGE_KEY = "POST_PUBLISH_CACHE_MAX_AGE";
-    private static final String POST_PUBLISH_CACHE_EXPIRY_OFFSET_KEY = "POST_PUBLISH_CACHE_EXPIRY_OFFSET";
     private static final String POST_PUBLISH_MICRO_CACHE_ENABLED_KEY = "POST_PUBLISH_MICRO_CACHE_ENABLED";
```

Comment:
This can go too.

Reply:
POST_PUBLISH_MICRO_CACHE_ENABLED_KEY now deleted